enterprise/reporting/event_sampler/badger_event_sampler.go

```diff
@@ -0,0 +1,153 @@
+package event_sampler
```

Comment:
add tests for both badger samples and in memory samplers

Reply:
these are present in `event_sampler_test.go`